src/libmongoc/tests/unified/runner.c

```diff
@@ -373,10 +379,13 @@ test_runner_new (void)
    callbacks = mongoc_apm_callbacks_new ();
    mongoc_apm_set_topology_changed_cb (callbacks, on_topology_changed);
    uri = test_framework_get_uri ();
-   /* Always use multiple mongos's if speaking to a mongos.
-    * Some test operations require communicating with all known mongos */
-   if (!test_framework_uri_apply_multi_mongos (uri, true, &error)) {
-      test_error ("error applying multiple mongos: %s", error.message);
+   /* In load balanced mode, the internal client must use the SINGLE_LB_MONGOS_URI. */
+   if (!test_framework_is_loadbalanced ()) {
+      /* Always use multiple mongos's if speaking to a mongos.
```

Comment:
My suspicion is that "mongoses" is probably correct (though odd). There must be a definitive answer! ;-)
https://www.merriam-webster.com/words-at-play/what-happens-to-names-when-we-make-them-plural-or-possessive

Reply:
"mongoses" is also used in specs. Consider the `useMultipleMongoses` test syntax.

Reply:
Updating to mongoses.